### PR TITLE
[HOPSWORKS-2504] feature description

### PR DESCRIPTION
**Pull request by o-alex**

### CODE REVIEW

File: templates/default/expat-site.xml.erb [hunk 1]
```diff
@@ -61,13 +61,27 @@
...
+        <snapshot>
+            <repo>
+                <name></name>
+                <location></location>
+            </repo>
+            <name></name>
+            <!-- repeated section for multiple indices -->
+            <indices></indices>
+            <ignoreUnavailable></ignoreUnavailable>
+        </snapshot>
```
Comment (maismail) [inline templates/default/expat-site.xml.erb:73]: Is it needed by any of the steps?

File: templates/default/expat-site.xml.erb [hunk 2]
```diff
@@ -61,13 +61,27 @@
...
     </elastic>
 
     <hops>
         <client>
             <user><%= node['hops']['hdfs']['user'] %></user>
         </client>
     </hops>
+
+    <epipe>
+        <path></path>
```
Comment (maismail) [inline templates/default/expat-site.xml.erb:83]: can you add the epipe base dir variable to the template

**Review by maismail — CHANGES_REQUESTED** [review]

File: templates/default/expat-site.xml.erb (hunk 2 above)
Comment (o-alex) [reply templates/default/expat-site.xml.erb]: fixed

File: templates/default/expat-site.xml.erb (hunk 1 above)
Comment (o-alex) [reply templates/default/expat-site.xml.erb]: fixed this.

**Review by maismail — APPROVED** [review]

**Pull request merged**